src/test/java/hudson/plugins/emailext/plugins/content/FailedTestsContentTest.java

```diff
@@ -18,9 +18,8 @@
 import java.util.List;
 
 import static java.util.Collections.singletonList;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
+
+import static org.junit.Assert.*;
```

Comment:
Please maintain the non * imports. Your IDE may have done this for you, I prefer to have the imports listed specifically.

Reply:
I removed it and retained the non * imports. I dont know how it came up again. Thanks for pointing out.